python/examples/13-agent-name-service/agent1.py

```diff
@@ -19,19 +23,25 @@ class Message(Model):
     endpoint=["http://localhost:8001/submit"],
 )
 
-ledger = get_ledger()
+fund_agent_if_low(bob)
+
 my_wallet = LocalWallet.from_unsafe_seed("registration test wallet")
 name_service_contract = get_name_service_contract()
 DOMAIN = "agent"
 
-for wallet in [my_wallet, bob.wallet]:
-    fund_agent_if_low(wallet.address())
+faucet = get_faucet()
+AGENT_BALANCE = bob._ledger.query_bank_balance(my_wallet)
```

Comment:
How about adding a `balance()` method directly to the agent?

Reply:
Ok If we can assume that we could import testnet ledger from network. And yeah incorrect variable name, should have been WALLET_BALANCE